scraper/smashgg.py

```diff
@@ -21,12 +21,15 @@ def check_for_200(response):
     return response
 
 class SmashGGScraper(object):
-    def __init__(self, path):
+    def __init__(self, path, excluded_phases):
```

Comment:
not a big deal, but might make more sense to pass the included_phases? this way if you only want to scrape one bracket, you just pass its phase (as opposed to having to scrape all the other ids of brackets in the tournament and then including all but the one you want).